src/app/containers/InfoPage/index.js

```diff
@@ -12,10 +10,33 @@ function InfoPage() {
         <title>Thông tin</title>
       </Helmet>
       <div className={styles.content}>
-        <p className={styles.title}>Quy chế của kỳ thi</p>
-        <span className={styles.text}>
-          <Markdown source={md} escapeHtml={false} />
-        </span>
+        <div className={styles.title}>Hướng dẫn</div>
+        <Link to="/help/register">
+          <div className={styles.item}>
+            <div className={styles.img}>
+              <img src="/src/assets/images/item.png" alt="Item" />
+            </div>
+            <div className={styles.itemcontent}>
+              <div className={styles.itemtitle}>
+                <h2>Hướng dẫn đăng ký Beginner Free Contest 24</h2>
```

Comment:
Use the data from here: https://info.freecontest.xyz/h%C6%B0%E1%BB%9Bng-d%E1%BA%ABn